test.py

```diff
@@ -39,7 +39,7 @@
 from test.pylib.util import LogPrefixAdapter
```

Comment:
@alecco I miss the motivation for changing the name separator. :: against . is 2-4x less efficient, depending how you count (characters or dots). Dots are really easier to type and easier to remember. Why did you decide to replace the separator?

Reply:
It's more standard and pytest (and many other frameworks) use `test::case` (as @nyh pointed out).

I added `suite::test::name` for simplicity, because `suite.name::case` looks ugly IMO.

Reply:
Besides being used by pytest (which is the basis for most of our test frameworks - dtest, topology, cql-pytest, alternator, etc.), there also a bizarre fact (in my experience - please check again in case I made a mistake) that when you write in a **github** issue `suite::somefilename::sometestname`  it allows searching each individual component (e.g., "sometestname") which is useful when looking for duplicate issues, while if you write `suite.somefilename.sometestname` the components aren't searchable (at least in my experience).

Another issue is that at least in pytest, the test file name does include the extension (e.g., .py), so a dot can't be used as a separator.

Reply:
I think we should use suite.file.test, or suite/file/test, since this is less typing on an US-ASCII keyboard layout. But OK, I see I am outvoted. Let's make it :: if such is the popular consensu. I believe, however, in pytest, weirdly, it's both dots and semicolons, so at least let's please not do it, let's consistently use semicolons...

Reply:
"suite/file/test" is problematic when the "file" part can have subdirectories (as we have in cql-pytest, and I see no reason not to have in other directories as well).

What do you mean that pytest has "both dots and semicolons"? Where do you use dots in pytest?